ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -516,7 +514,7 @@ translateLedgerViewByronToShelleyWrapper =
 -------------------------------------------------------------------------------}
 
 translateLedgerStateShelleyToAllegraWrapper ::
-     (PraosCrypto c, DSignable c (Hash c EraIndependentTxBody))
+     (PraosCrypto c)
```

Comment:
As there is only one constraint I usually favor removing the parentheses although I'm not sure what the style guide says

Reply:
Seems like the style guide agrees with you :)